tests/robustness/multiclient_test/framework/harness.go

```diff
@@ -207,31 +208,39 @@ func (th *TestHarness) Engine() *engine.Engine {
 
 // Run runs the provided function asynchronously for each of the given client
 // contexts, waits for all of them to finish, and optionally cleans up clients.
-func (th *TestHarness) Run(ctxs []context.Context, cleanup bool, f func(context.Context)) {
-	var wg sync.WaitGroup
+func (th *TestHarness) Run(t *testing.T, ctxs []context.Context, cleanup bool, f func(context.Context, *testing.T)) {
```

Comment:
I suggest that `t` should be the second parameter. Go idiom is to have context as the first parameter - I think that should still apply even though there is a slice of contexts here.

Reply:
It throws a linter error (because it's a slice like you said). I will look to see if it's set up as an ignorable linter error though.

Reply:
This is the linter error, but I realized I can suppress it with `// nolint:thelper` so I'll go ahead and do that.

```console
$ make lint
/repovol/kopia/tools/.tools/golangci-lint-1.37.1/golangci-lint --deadline 300s run
tests/robustness/multiclient_test/framework/harness.go:211:24: parameter *testing.T should be the first or after context.Context (thelper)
func (th *TestHarness) Run(ctxs []context.Context, t *testing.T, cleanup bool, f func(context.Context, *testing.T)) {
                       ^
Makefile:57: recipe for target 'lint' failed
make: *** [lint] Error 1

```